experiment_buddy/experiment_buddy.py

```diff
@@ -234,17 +238,39 @@ def _setup_tb(logdir):
     return tensorboardX.SummaryWriter(logdir=logdir)
 
 
-def _ensure_scripts(extra_headers):
+def _ssh_session_connected():
```

Comment:
a better name would be
_open_ssh_session()
the fact the session is connected is expected by the name